nexus/src/app/instance.rs

```diff
@@ -378,15 +403,11 @@ impl super::Nexus {
     pub async fn instance_start(
         &self,
         opctx: &OpContext,
-        organization_name: &Name,
-        project_name: &Name,
-        instance_name: &Name,
+        authz_instance: &authz::Instance,
     ) -> UpdateResult<db::model::Instance> {
         let (.., authz_instance, db_instance) =
             LookupPath::new(opctx, &self.db_datastore)
-                .organization_name(organization_name)
-                .project_name(project_name)
-                .instance_name(instance_name)
+                .instance_id(authz_instance.id())
```

Comment:
Same here as above -- we already did a lookup in the caller.  It'd be nice to just use that here instead of doing another lookup.  (You could even wind up with a different result here than the caller got.)